scripts/run-build.ps1

```diff
@@ -72,7 +72,7 @@ $env:PATH = "$env:DOTNET_INSTALL_DIR;$env:PATH"
 # Restore the build scripts
 Write-Host "Restoring Build Script projects..."
 pushd $PSScriptRoot
-dotnet restore --disable-parallel
+dotnet restore --disable-parallel --infer-runtimes
```

Comment:
cc @pakrym build is breaking without this, did you forget to add it here is there a different issue?